fabulaws/library/wsgiautoscale/api.py

```diff
@@ -466,7 +466,12 @@ def _new(deployment, environment, role, avail_zone=None, count=1, **kwargs):
                      tags=tags, volume_size=vol_size, volume_type=vol_type,
                      deploy_user=env.deploy_user, security_groups=sec_grps,
                      **extra_args)
-        server.setup()
+        try:
+            server.setup()
+        except:
+            if terminate_on_failure:
+                server.terminate()
```

Comment:
Can we log or output whatever the exception is that's leading us to terminate this server?